spec/unit/block_score/actions/delete_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+RSpec.describe BlockScore::Actions::Delete do
+  describe '#delete' do
+    subject(:candidate) { create(:candidate) }
+    before { candidate.delete }
+
+    it { is_expected.not_to be_persisted }
+    it { expect(candidate.deleted).to be true }
```

Comment:
As would I and I have done (need to check to see if predicate is public). But the scary part is it's not immutable.